tests/perftest_compilers_single_value.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# This script assumes that there is only one x-value (e.g., the
+# perftest does not check multiple buffer sizes).  In particular,
+# it was created for md5, sha1, and sha256.
```

Comment:
This file resulted from adjusting the `tests/md5/perftest.sh` script for the new `make perftests`, and then I realized that I could generalize it to also handle sha1 and sha256, which also contain that file (but probably just because we copied from md5 when we created those directories).

I'm not certain if it's worth keeping the file around, or modifying it, and even if we modify it, I'm not certain that it belongs in the same PR as `make perftests`.